app/src/main/java/com/javahelp/frontend/MainActivity.java

```diff
@@ -48,7 +48,15 @@ public void onClick(View view) {
             }
         });
 
-
+        // link the button to the second activity(front page)
+        buttoncr.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent intent = null;
```

Comment:
Could these two be combined into one step?

Reply:
The one above and below this message. Sorry this splits them apart. 
Like  `Intent intent = new Intent(MainActivity.this,ClientRegistrationActivity.class);`. Is there any specific reason behind separating these two?

Reply:
I am bit confused. Can you explain more?

Reply:
Ah wait, I see what you mean. I can combine them.

Reply:
I think they mean the same, and I think both ways are ok